src/app/containers/RadioPageMain/index.jsx

```diff
@@ -37,6 +37,7 @@ const RadioPageMain = ({ pageData }) => {
           group5: 20,
         }}
         enableGelGutters
+        style={{ flexGrow: '1' }}
```

Comment:
Bit less certain here, but I think you should also use 
```javascript
const StyledGelPageGrid = styled(GelPageGrid)`flexGrow: '1'` 
```
Then replace line 27 with `<GelPageGrid>`

Again to style with the convention of using styled components